source/documentation/team-information/internal-processes/assess-reserved-instances.html

```diff
@@ -0,0 +1,85 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Assessment and Purchase of Reserved Instances
+last_reviewed_on: 2025-05-06
+review_in: 3 months
+---
+
+# Assessment and Purchase of Reserved Instances (RI)
+
+## Purpose
+
+The purpose of this guide is to formalise the process for assessment and purchase of RIs. A Reserved Instance is a cloud pricing model where users commit to using specific resources for a set term in exchange for discounted rates.
+
+The guide provides a reusable framework to maximise effective use of RIs, and avoid vacancy (wasted commitments).
+
+## Assessment of new RI opportunities
+
+There are a number of steps to gather data and evaluate RIs to ensure that we are purchasing the best opportunities and minimise risk of waste.
+
+### Gather recommendations
+
+Use AWS [Reserved Instances Recommendations](https://docs.aws.amazon.com/cost-management/latest/userguide/ri-recommendations.html) via the Billing and Cost Management Console, to indentify potential RI optimisation opportunities.
+
+Depending on the type of resources there are some filters that can be applied to identify appropriate recommendations:
+
+- **Offering class** - This is EC2 specific and allows you to select **"[Convertible](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ri-convertible-exchange.html)"** RIs. This can help mitigate risk of waste by converting the RI to a different type.
+- **Recommendation level** - Select **"Payer"**. This allows us to purchase and use RIs across all accounts within the MoJ Organisation.
+- **Term** - Select **"1-year"**. We do not purchase RIs over a year to mitigate risk of waste. We can consider this is the future as our FinOps practice matures.
```

Comment:
consider this 'is' the future - should that be 'in'?

Reply:
Typo fixed